tests/migrations/test_runner.py

```diff
@@ -358,6 +358,23 @@ def get_total_migration_count() -> int:
     return count
 
 
+@pytest.mark.clickhouse_db
+def test_get_active_migration_groups() -> None:
+    settings.SKIPPED_MIGRATION_GROUPS = {"search_issues"}
+    active_groups = get_active_migration_groups()
+    assert (
+        MigrationGroup.SEARCH_ISSUES not in active_groups
+    )  # should be skipped by SKIPPED_MIGRATION_GROUPS
+
+    settings.READINESS_STATE_MIGRATION_GROUPS_ENABLED = {"search_issues"}
+    active_groups = get_active_migration_groups()
+    assert (
+        MigrationGroup.SEARCH_ISSUES in active_groups
+    )  # should be active by readiness_state
+
+    importlib.reload(settings)
```

Comment:
It's good practice to use a fixture for stuff like this in case your test throws (since you are affecting global state) for example:

```python3
@pytest.fixture(scope="function")
def temp_settings():
    from snuba import settings
    yield settings
    importlib.reload(settings)
    
def test_get_active_migration_groups(temp_settings) -> None:
    pass    
```

Reply:
Updated 👍